cri/v1alpha2/cri.go

```diff
@@ -916,44 +916,7 @@ func (c *CriManager) ContainerStatus(ctx context.Context, r *runtime.ContainerSt
 		})
 	}
 
-	// Interpret container states.
-	var state runtime.ContainerState
-	var reason, message string
-	if container.State.Status == apitypes.StatusRunning {
-		// Container is running.
-		state = runtime.ContainerState_CONTAINER_RUNNING
-	} else {
-		// Container is *not* running. We need to get more details.
-		//		* Case 1: container has run and exited with non-zero finishedAt time
-		//		* Case 2: container has failed to start; it has a zero finishedAt
-		//				  time, but a non-zero exit code.
-		//		* Case 3: container has been created, but not started (yet).
-		finishTime, err := time.Parse(utils.TimeLayout, container.State.FinishedAt)
```

Comment:
PouchContainer will promise the consistency about `FinishedAt`, `CreatedAt`, `StartedAt` and container state